### PR TITLE
experiment.BehaviorTrial {Auto,Free}Water  

### PR DESCRIPTION
**Pull request by ixcat**
Add support for BehaviorTrial.{auto_water,free_water}.

### CODE REVIEW

File: pipeline/ingest/behavior.py [hunk 1]
```diff
@@ -466,6 +466,11 @@ def make(self, key):
                 outcome = 'ignore'
 
             bkey['outcome'] = outcome
+
+            # Determine free/autowater
+            bkey['auto_water'] = True if gui['Autowater'][0] == 2 else False
```
Comment (ttngu207) [inline pipeline/ingest/behavior.py]: Shouldn't this be: 
`bkey['auto_water'] = True if gui['Autowater'][0] == 1 else False`

Comment (ixcat) [reply pipeline/ingest/behavior.py]: per discussion with susu, 2 is enabled and 1 is disabled- not sure why these values were chosen.

Comment (ixcat) [reply pipeline/ingest/behavior.py]: will make a comment.

Comment (ttngu207) [reply pipeline/ingest/behavior.py]: I think it's the other way around:
``` When SessionData.TrialSettings.GUI.Autowater = 1, autowater on, when SessionData.TrialSettings.GUI.Autowater = 2, autowater off.```

**Pull request merged**